static/.htaccess

```diff
@@ -306,6 +298,50 @@ RewriteRule ^docs/0.25/components/zeebe/tasklist-user-guide /docs/0.25/component
 # redirect best-practices overview page
 RewriteRule ^docs/components/best-practices/?$ /docs/components/best-practices/overview/ [R=301,L]
 
+# optimize docs get their own section of the site
+## these rules previously existed, but are now updated to point at the /optimize section:
+RewriteRule ^docs/components/optimize/userguide/combined-reports/(.*)$ /optimize/userguide/combined-process-reports/$1 [R=301,L]
+RewriteRule ^docs/components/optimize/userguide/process-analysis/overview/(.*)$ /optimize/userguide/process-analysis/process-analysis-overview/$1 [R=301,L]
+RewriteRule ^docs/components/optimize/userguide/process-analysis/overview$ /optimize/userguide/process-analysis/process-analysis-overview/ [R=301,L]
+RewriteRule ^docs/components/optimize/userguide/decision-analysis/overview/(.*)$ /optimize/userguide/decision-analysis/decision-analysis-overview/$1 [R=301,L]
+RewriteRule ^docs/components/optimize/userguide/decision-analysis/overview$ /optimize/userguide/decision-analysis/decision-analysis-overview/ [R=301,L]
+RewriteRule ^docs/components/optimize/userguide/additional-features/filters/(.*)$ /optimize/userguide/process-analysis/filters/$1 [R=301,L]
+## these rules didn't exist, but now point anything in docs/components/optimize to /optimize:
+RewriteRule ^docs/components/optimize/what-is-optimize(\/?)$ /optimize/what-is-optimize/ [R=301,L]
```

Comment:
Most of the redirects in this file have two rules per URL. In a few cases it's to handle child routes, but in the large majority _I think_ it's to handle two cases each: when the URL contains a trailing slash, and when it doesn't. 

These new rules only contain one line per URL, but the `(\/?)` at the end of each catches both cases -- trailing slash or no trailing slash. 

If I'm missing another reason for the dual rules, let me know!

(Here's an example of another rule in the file, which uses two sub-rules): 

```
RewriteRule ^docs/self-managed/optimize-deployment/setup/configuration(.*)$ /docs/self-managed/optimize-deployment/configuration/system-configuration [R=301,L]
RewriteRule ^docs/self-managed/optimize-deployment/setup/configuration$ /docs/self-managed/optimize-deployment/configuration/system-configuration [R=301,L]
```

Reply:
> when the URL contains a trailing slash, and when it doesn't.

Nailed it. Child routes and trailing slashes. 

We had to go back and forth a bit on trailing vs. no trailing slashes. I think we've committed to trailing slashes at this point so if you have a good way to reduce these to ONE rule, I'm all for it.

Reply:
Plus I'm terrible at anything regex or regex leaning.